accounting/account/admin.py

```diff
@@ -23,7 +23,17 @@
 
 from django.contrib import admin
 
-from accounting.account.models import *  # pylint: disable=wildcard-import,unused-wildcard-import
+from accounting.account import models
+from accounting.common.admin import UuidModelAdmin
```

Comment:
Well, `UUIDField` is an upstream Django convention, and we don't have to necessarily follow that. It's also only used in one place downstream (`accounting/common/models.py`), so not sure if we would change all that just for this. I also generally stick with not uppercasing these things, because imagine what happens when you have to concatenate two acronyms: `APIUUIDModel`. Surely `ApiUuidModel` is easier to parse. And with an IDE this should be made pretty simple I hope.

Reply:
@UmanShahzad Alright, you convinced me 😉